CODEOWNERS

```
@@ -12,6 +12,6 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-* @lquerel @jmacd @moh-osman3 @codeboten
+* @lquerel @jmacd @moh-osman3 @codeboten @drewrelmas
```

@jmacd up to you, but you could also use the group here to avoid needing these changes in the future. From Collector-Contrib:

> cmd/builder/                             @open-telemetry/collector-approvers

---

I looked at other examples and I wasn't sure that would work.
```suggestion
*  @open-telemetry/collector-approvers
```